configs/eln_extras_meta.yaml

```diff
@@ -54,20 +54,20 @@ data:
     - pam_krb5
     - parallel
     - pngquant
+    - poi-tracker
     - pore
     - pv
     - python3-pystemd
     - python3-testslide
     - python3-zstd
-#    - rpmdistro-repoquery
```

Comment:
This was disabled temporarily due to FTI; see https://pagure.io/rpmdistro-repoquery/pull-request/23

This should probably remain (albeit commented) until that is fixed.

Reply:
rpmdistro-repoquery has been fixed and uncommented; this will need to be rebased but now that won't be lost either.

Reply:
> rpmdistro-repoquery has been fixed and uncommented; this will need to be rebased but now that won't be lost either.

ping @michel-slm

Reply:
Ack, I need to add another package too. Will update after the weekend, last two weeks were PTO+buried in work

Reply:
done